File: trail1/별표 출력하기 11/print-star-11.py
N = int(input())
size = 2 * N + 1
board = [['*' for _ in range(size)] for _ in range(size)]
for i in range(size):
    for j in range(size):
        if i % 2 == 1 and j % 2 == 1:
            board[i][j] = ' '

for i in range(size):
    for j in range(size):
        print(board[i][j], end=' ')
    print()

File: trail1/별표 출력하기 11/print-star-11.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);
    int size = 2*N+1;
    char board[size][size];
    for (int i=0; i<size; i++) {
        for (int j=0; j<size; j++) {
            if (i % 2 == 1 && j % 2 == 1) {board[i][j] = ' ';}
            else {board[i][j] = '*';}
        }
    }
    for (int i=0; i<size; i++) {
        for (int j=0; j<size; j++) {
            printf("%c ", board[i][j]);
        }
        printf("\n");
    }
    return 0;
}